app/controllers/users_controller.rb

```diff
@@ -0,0 +1,61 @@
+class UsersController < ApplicationController
```

Comment:
This is the controller that is not required, as Devise takes care of all of these actions for you